menu_technical_info/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# © 2016 Onestein (<http://www.onestein.eu>)
```

Comment:
replace `©` by `Copyright` according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L2 (everywhere)